tools/db_crashtest.py

```diff
@@ -114,7 +114,7 @@
     "use_direct_reads": lambda: random.randint(0, 1),
     "use_direct_io_for_flush_and_compaction": lambda: random.randint(0, 1),
     "mock_direct_io": False,
-    "cache_type": lambda: random.choice(["fast_lru_cache", "lru_cache"]),   # clock_cache is broken
+    "cache_type": "lru_cache", #lambda: random.choice(["fast_lru_cache", "lru_cache"]),   # clock_cache is broken
```

Comment:
When "why" is not obvious, it's important to include. Why is fast_lru_cache disabled? It's not necessarily important to go into all the details, but at least make it clear what is intentional and what might be an accidental omission or accidentally included in a pull request. e.g.

```
  # fast_lru_cache currently incompatible with crash test; clock_cache is broken
```